src/run.js

```diff
@@ -456,6 +456,10 @@ const minimalcss = async options => {
           return;
         }
 
+        if (!node.prelude.children) {
+          throw new Error(`Invalid CSS found: ${node.prelude.value}`);
```

Comment:
We can use `href` so it would be easy to understand where the error comes from

Reply:
how about:
```js
throw new Error(
  `Invalid CSS found while evaluating ${href}: "${
    node.prelude.value
  }"`
);
```
?

Reply:
Maybe `SyntaxError at ${href} near "${node.prelude.value}"`, but have no strong opinions